templates/leanproject.md

```diff
@@ -6,7 +6,7 @@ Everything is done using the `leanproject` command-line tool. You can
 use `leanproject --help` to get the list of available commands and
 options.
 
```

Comment:
```suggestion

`leanproject` only supports Lean 3. If you are using Lean 4, the information 
on this page is not relevant.

```
?

We should (separately) take a look over the website and clarify the Lean 3/4 language wherever needed.

Reply:
Agreed: see also #157.